packages/backend/src/nest/storage/certificates/certificates.store.ts

```diff
@@ -41,6 +41,7 @@ export class CertificatesStore extends EventEmitter {
         write: ['*'],
       },
     })
+    await this.store.load()
```

Comment:
Do you know what effect fetchEntryTimeout has? Is there a default fetchEntryTimeout? I can look these things up too, but curious if you've looked into it.

Reply:
I think the default is to wait until it loads.  If we think there will be hanging we can add a timeout back.

Reply:
Interesting, perhaps the timeout could have been causing issues